images/scanner-test.Dockerfile

```diff
@@ -42,8 +42,8 @@ RUN dnf install -y \
         expect \
         gcc \
         gcc-c++ \
-        google-cloud-sdk \
-        google-cloud-sdk-gke-gcloud-auth-plugin \
+        google-cloud-cli \
```

Comment:
you already did this :) https://github.com/stackrox/rox-ci-image/pull/200/files#diff-f849cde71446382ccc46e79ca6640ed1aec3c28388622eb99e61e339c7a70a91R45

may need to rebase